src/style.css

```diff
@@ -1,3 +1,64 @@
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav__list {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 50px;
+}
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  margin-left: 20px;
```

Comment:
There shouldn't be the margin in the first element:
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/f55e29ef-3f82-483e-8031-5ab6c27446ac)

That's the reason tests fail